### PR TITLE
style: improve style mermaid diagrams light/dark mode

### PR DESCRIPTION
**Pull request by houssemmimoun27**
This PR enhance the syle for `mermaid diagrams` for dark and light mode by rendering the colors defined by `OKP4/UI`.
<details>

Screenshot 2023-08-25 at 14 14 51

Screenshot 2023-08-25 at 14 15 07

<summary> Screenshots </summary>

### CODE REVIEW

File: src/scss/main.scss [hunk 1]
```diff
@@ -179,6 +179,10 @@ html {
 }
 
 svg {
+  path {
+    stroke: var(--ifm-graph-base-line-color) !important;
```
Comment (fredericvilcot) [inline src/scss/main.scss:183]: @houssemmimoun27 
By doing this, you  say you want to apply to all `svg` paths a color defined by this variable.

But, is this what we want? I'm not sure.

As an example, this line of code has modified the burger menu icon:

Before:
Capture d’écran 2023-08-30 à 17 55 50

After:
Capture d’écran 2023-08-30 à 17 56 18

Please be careful, and don't forget that the `!important` instruction overrides any style with any context in any stylesheet.

File: src/scss/main.scss [hunk 2]
```diff
@@ -214,6 +218,34 @@ svg {
...
+  text.actor > tspan,
+  .messageText {
+    fill: var(--ifm-menu-color) !important;
+  }
+
+  g {
+    .actor,
+    circle {
+      fill: var(--ifm-graph-base-actor-color) !important;
+      stroke: var(--ifm-graph-base-actor-color) !important;
```
Comment (fredericvilcot) [inline src/scss/main.scss:232]: Same here, style from all circles included in the group will be modified

File: src/scss/main.scss [hunk 3]
```diff
@@ -214,6 +218,34 @@ svg {
...
+
+    .noteText > tspan {
+      fill: var(--ifm-graph-base-rect-text-color) !important;
+    }
+  }
+
+  line {
+    fill: var(--ifm-graph-base-line-color) !important;
+    stroke: var(--ifm-graph-base-line-color) !important;
+  }
```
Comment (fredericvilcot) [inline src/scss/main.scss:248]: Same here

**Review by fredericvilcot — CHANGES_REQUESTED** [review]

**Review by ccamel — APPROVED** [review]
LGTM!

**Review by fredericvilcot — APPROVED** [review]
Looks better, thanks

**Comment by fredericvilcot** [comment]
@MdechampG 
Is this PR ok for you?

**Review by MdechampG — CHANGES_REQUESTED** [review]
It is working on the mermaid shown in screenshot but not on the ones in the part Technical documentation/Ontology/Ontology principles. 

image

**Comment by ccamel** [comment]
As a result of #371, the latest updates to the [Mermaid](https://mermaid.js.org/) plugin, along with the newest version v3 of [Docusaurus](https://docusaurus.io/) which is now configured with the [@docusaurus/theme-mermaid](https://docusaurus.io/docs/next/api/themes/@docusaurus/theme-mermaid) theme, have enhanced the compatibility of mermaid diagrams for both dark and light modes. Given this upgrade, which effectively renders this PR redundant, I recommend its closure. Should there be a need for cosmetic modifications, a new ticket can be opened and addressed, keeping in mind these recent updates.

fyi @MdechampG

**Pull request closed without merging**